src/nyc_trees/apps/survey/fixtures/species.json

```diff
@@ -1278,12 +1265,12 @@
         "model": "survey.Species",
         "fields": {
             "species_code": "PR",
-            "scientific_name": "Prunus serrulata",
+            "scientific_name": "Prunus cultivar",
             "created_at": "2113-04-10T17:11:50.350Z",
             "updated_at": "2113-04-10T17:11:50.350Z",
             "forms_id": 92410,
             "common_name": "Cherry",
-            "cultivar": "Green leaf"
+            "cultivar": ""
```

Comment:
The row in Species_List_rev3.xlsx with forms_id 92410 still has a cultivar value of "Green Leaf". Are we dropping it here for a specific reason?